src/languages/regex.js

```diff
@@ -0,0 +1,101 @@
+/*
+Language: regex
+Description: Regular expressions 
+Author: Konrad Rudolph <konrad.rudolph@gmail.com>
+*/
+
+export default function (hljs) {
+  const ESCAPE_SEQUENCE = {
+    className: 'built_in',
+    begin: [
+      // Unicode script/block/category
+      /\\[pP](?:[^{]|\{\^?[^}]+\})/,
+      // Unicode character
+      /\\[ux](?:[0-9a-fA-F]{1,4}|\{[0-9a-fA-F]{1,4}\})/,
+      // Backreferences
+      /\\k(?:'-?\d+'|<-?\d+>|{-?\d+})/,
+      /\\g(?:'-?\d+'|<-?\d+>|{-?\d+}|-?\d+)/,
+      // Everything else
+      /\\./
+    ].map(r => r.source).join('|')
+  };
+
+  const CHARACTER_CLASS = {
+    className: 'string',
```

Comment:
`regex.character_class`? (I wonder if we should care about abbreviations with modern gzip and the fact that most web traffic today should be at least gzipped...